fe/fe-core/src/test/resources/data/es/es7_aliases_mapping.json

```diff
@@ -0,0 +1,50 @@
+{
+  "test2_202207": {
+    "mappings": {
```

Comment:
Where is `aliases` field?

Reply:
My problem, I'm going to replace it

Reply:
`JSONObject test2Aliases = EsUtil.getMappingProps("test2", loadJsonFromFile("data/es/es7_aliases_mapping.json"),
                null);` _mapping not contains aliases, I use test2 to get real index.